partner/WHMCS/README.md

```diff
@@ -0,0 +1,28 @@
+# WHMCS Jetpack Module
+
+This module is intended to assist Jetpack Hosting partners in managing Jetpack plans for their users. The module allows a host to create a product from the WHMCS admin that can used to provision a Jetpack plan at checkout and manage the plan from the Clients area in WHMCS when needed. Module functions include both automatic provisioning on checkout as well as manual provisioning and plan cancellation from the WHMCS Clients management. 
+
+### Getting Started:
+This documentation includes information for managing the WHMCS module. For documentation on the Jetpack Hosting Partners Program please refer to the [Jetpack Hosting Partner Docs][jetpack partner docs]. If you’d like to become a Jetpack Hosting Partner please visit https://jetpack.com/for/hosts/ for more information and to get started. In order to use the module a valid WHMCS license as well as a Jetpack Hosting Partner account are required. To get started with WHMCS please visit https://www.whmcs.com/. 
+
+### Installation/Setup:
+To install the module upload the ```/jetpack``` folder to the ```/modules/servers``` directory of your WHMCS installation. A server is not required for the module to function so the module will not be listed in the WHMCS Servers under the Setup tab for Products/Servers. Once uploaded to WHMCS the module is ready for use and can be set up when making a new product in WHMCS.
```

Comment:
When I initially ran through these steps, I attempted to symlink `/modules/servers/jetpack` to the `/jetpack/partner/WHMCS/modules/servers/jetpack` directory, but the module wouldn't show. So, I then actually copied the module directory from Jetpack to WHMCS and things worked. 

There's always a chance I symlinked things incorrectly. But, I figured I'd mention it in case that's a known issue.

Reply:
Thanks for the feedback! I'll definitely take a look in to this and if there are potentially issues include them in the docs.